math/wxMaxima/Portfile

```diff
@@ -1,59 +1,109 @@
-PortSystem      1.0
-PortGroup       active_variants 1.1
-PortGroup       wxWidgets 1.0
-
-name            wxMaxima
-version         15.04.0
-revision        0
-maintainers     nomaintainer
-license         gpl
-platforms       darwin
-categories      math aqua
-description     Graphical user interface for Maxima
-long_description \
-                wxMaxima is a cross platform GUI for the computer algebra \
-                system maxima based on wxWidgets.
-homepage        http://wxmaxima.sourceforge.net/
-master_sites    sourceforge:project/[string tolower ${name}]/${name}/${version}
-livecheck.regex (?:[string tolower ${name}]|${name})-(\[a-z0-9.\]+)${extract.suffix}
-checksums       rmd160  04295f10ccd074d7069d00b1a0fe41884817687c \
-                sha256  0005029798703470333309fda29c731e79c85de1c7fc09b16cb87045a03aa4ba
-
-# Upstream changed the distname's spelling some time after 13.x.
-distname        wxmaxima-${version}
-
-wxWidgets.use   wxWidgets-3.0
-
-patchfiles      patch-src_main.cpp.diff \
-                patch-src_Dirstructure.h.diff \
-                patch-src_wxMaxima.cpp.diff
-
-depends_lib     port:libiconv \
-                port:libsdl \
-                port:libxml2 \
-                port:${wxWidgets.port} \
-                port:zlib
-depends_run     port:maxima
-
-require_active_variants gnuplot wxwidgets
-
-post-patch {
-    reinplace "s|@@PREFIX@@|${prefix}|" ${worksrcpath}/src/main.cpp
-}
+PortSystem          1.0
 
-depends_skip_archcheck  maxima
-configure.args  --enable-printing --with-wxdir=${wxWidgets.wxdir}
-build.target-append wxMaxima.app
+PortGroup           active_variants 1.1
+PortGroup           wxWidgets 1.0
 
-destroot {
-    xinstall -m 755 -d ${destroot}${applications_dir}
-    file copy ${worksrcpath}/wxMaxima.app ${destroot}${applications_dir}
+name                wxMaxima
+version             18.02.0
+revision            0
+maintainers         nomaintainer
+license             gpl
+# openssl is a dependency of curl in cmake
+license_noconflict  openssl
+platforms           darwin
+categories          math aqua
+description         Graphical user interface for Maxima
+long_description    wxMaxima is a cross platform GUI for the computer algebra \
+                    system maxima based on wxWidgets.
+homepage            http://andrejv.github.io/wxmaxima/
+
+variant devel description {Get a development version with git.} {}
```

Comment:
We don't allow variants to change the version number. So remove the devel variant and create a -devel sub-port instead.

Reply:
This is probably true of the Maxima pull request, https://github.com/macports/macports-ports/pull/2033 as well

Reply:
Thanks for the comment.
I'm not familiar with making a portfile.
So I don't know how to make a sub-port and how to debug it.
I read the "MacPorts Guide", but I could not find it.
Please let me know.

Reply:
Basically you make a new block in the portfile with the new information for the devel port. In this case you will create a block like the following. See the uhd portfile as an example. 

```
subport wxMaxima-devel {

}
```

Reply:
Thanks for the info.
It is not easy for me to check 'sub-port'.
But I finally understood that 'portindex' is needed.
I need some more time to check 'sub-port'.